pact/verify_wrapper.py

```diff
@@ -169,6 +169,7 @@ def call_verify(self, *pacts, provider_base_url, provider, **kwargs):
 
         if(kwargs.get('verbose', False) is True):
             command.extend(['--verbose'])
+        command.extend(self._get_enable_pending_flag(kwargs.get('enable_pending')))
```

Comment:
_get_boolean_flag('pending')

Reply:
>There is no such a flag as --no-verbose, and for just one case I think it is a premature optimisation.

Reply:
Maybe I am wrong, but I think there is a `no-verbose` flag.
https://github.com/pact-foundation/pact-python/pull/171/files#diff-99dbf564da4f139a45a61bf9c28b4f28R104

Reply:
It is lacking in the ruby verifier help message (see https://github.com/pact-foundation/pact-provider-verifier/blob/master/README.md), but I checked and indeed this ruby binary accepts --no-verbose. Although according to https://github.com/pact-foundation/pact-python/pull/171#discussion_r483937714 I'll drop the negative flag passing.